base/Dockerfile

```diff
@@ -18,12 +18,12 @@ ADD https://github.com/AcalephStorage/kviator/releases/download/v${KVIATOR_VERSI
 # Download confd
 ADD https://github.com/kelseyhightower/confd/releases/download/v${CONFD_VERSION}/confd-${CONFD_VERSION}-linux-amd64 /usr/local/bin/confd
 
-# Install prerequisites
-RUN apt-get update &&  apt-get install -y wget unzip uuid-runtime && \
+# install prerequisites
+RUN apt-get update &&  apt-get install -y wget unzip uuid-runtime python-setuptools udev && \
```

Comment:
I take it udev is a requirement somewhere?  The idea of installing udev in a container is... not pleasant.